test/web.test.ts

```diff
@@ -0,0 +1,32 @@
+import {Webpage} from "../src/Webpage";
+import * as Server from "mock-http-server"; 
+import * as fs from "fs";
+
+const web: Webpage = new Webpage;
+const server: Server = new Server({host: "localhost", port: 9000});
+const url: string = "http://localhost:9000/resource";
```

Comment:
Just a side note: if you are instantiating something like a class or allocating a string, most of the times you won't have to specify the type. Typescript is smart enough to infer it by itself